src/defaultSettings.js

```diff
@@ -2359,6 +2359,19 @@ DefaultSettings.prototype = {
    */
   preventOverflow: false,
 
+  /**
+   * Prevents wheel event on overlays for doing default action.
+   *
+   * @type {Boolean}
+   * @default false
+   *
+   * @example
+   * ```js
+   * preventWheel: false,
```

Comment:
Please check if is it possible to mark it as private. What's more please add this to the TypeScript definition file.